### ISSUE TITLE
Add support for ntfy.sh

### ISSUE THREAD

**Issue by jonasehrlich** [jonasehrlich/ek-scraper#30]

**Comment by vitezfh** [comment]
I really want to help with this, but my `poetry install` is failing followed by this message:

```
...
...

Note: This error originates from the build backend, and is likely not a problem with poetry but with frozenlist (1.3.3) not supporting PEP 517 builds. You can verify this by running 'pip wheel --no-cache-dir --use-pep517 "frozenlist (==1.3.3)"'.
```

Any tips here?
I saw a pretty old issue on poetry's github where people still don't know why this occasionally happens or how to solve it. 

Thank you for this amazing, clean tool!!

**Comment by jonasehrlich** [comment]
I have never seen this issue. Do you have any info on your environment?

**Comment by jonasehrlich** [comment]
Update: I just re-installed the environment and did not see the issue. 

Feel free to send a PR to add support for ntfy.sh, I currently don't really have a need for it, so I just added this issue as a reminder.

**Comment by jonasehrlich** [comment]
Opened a jehrlich/ntfy.sh branch with an initial implementation. Somehow there are still issues with the delivering of these notifications at least on iOS. The notifications are visible in the ntfy app, but no push notifications are created.

**Comment by jonasehrlich** [comment]
This is merged and available in v0.2.1 on PyPI

**Comment by vitezfh** [comment]
I was away on vacation and wanted to get back to this. 
Testing it with ntfy right now and it seems to work like a charm!

TYSM!!
